### DISCUSSION TITLE
additional info

### DISCUSSION THREAD

**Discussion by ClashSAN** [sdwebui-w-horde/sd-webui-stable-horde-worker#24] [General]
Added an additional info section to the extension wiki. Please see if it's suitable and accurate: https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Extensions#stable-horde-worker

Are more exotic settings, such as clip skip, vaes, applied for end-user when set in webui?
What samplers can an end-user have?

**Comment by MaikoTan** [comment]
Thanks for listing my extension in the wiki!
I think it is correct.

I make the extension itself not to affect the WebUI user (aka. the user who provides workers) too much. As the VAE settings, the Clip Skip, the Hypernetworks and so on are configured in the settings page instead of the txt2img page, of course, those settings would affect the generated image that is requested by the end users.

As for samplers, currently the Stable Horde provides 11 samplers that you could found [here](https://github.com/db0/AI-Horde/blob/cd39da6fc396f55f85b94a98acd94a96736d2347/horde/apis/models/stable_v2.py#L38), that is also the extension supported right now.

By the way, I am now continuing to work on [the PR that provides a new tab page](https://github.com/sdwebui-w-horde/sd-webui-stable-horde-worker/pull/13) for the extension. It is hard work for me because I am not familiar with Python and Gradio, but I assume it can be done in the next week, may I send you a message to change the wiki content when I have done this?

current progress:
![image](https://user-images.githubusercontent.com/19927330/211752597-5b5c8aed-e17f-4120-aa76-107851c6d571.png)

**Reply by ClashSAN** [reply]
sure.

**Reply by ClashSAN** [reply]
what commit version are you currently testing with? I don't think its working on the latest versions. what would you like to put on the extensions page?

**Reply by MaikoTan** [reply]
I am always on the latest commit since I merged the tab page PR, at least it is working on my side now. Is there any problem with yours? Maybe you can try to update your webui to the latest commit since it changed the API recently, it that still does not work, you could consider opening an issue if you have found any bugs!

**Reply by MaikoTan** [reply]
Also, I have updated the readme for the setup guide, and I have invited you as the organisation members.

**Reply by MaikoTan** [reply]
Oops, I have found an error, sorry!

I fixed it in https://github.com/sdwebui-w-horde/sd-webui-stable-horde-worker/pull/53 , please update the version.
